tusclient/uploader.py

```diff
@@ -61,6 +63,17 @@ class Uploader(object):
         - retry_delay (int):
             How long (in seconds) the uploader should wait before retrying a failed upload attempt.
             If not specified, it defaults to 30.
+        - resumable (bool):
+            Determines whether or not url should be stored, and uploads should be resumed.
+        - storage (<tusclient.storage.interface.Storage>):
```

Comment:
What about naming this `urlStorage`? Or else, this may be confused with the upload storage which is not correct.